py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py

```diff
@@ -639,9 +637,11 @@ def build(self):
                  "--use_basic_auth={0}".format(self.use_basic_auth),
               ]
 
+      py3_template = argo_build_util.deep_copy(task_template)
+      py3_template["container"]["image"] = "gcr.io/kubeflow-ci/test-worker-py3:789005d"
```

Comment:
nit: Should we define a constant at the top of the file so its easy to change?

Reply:
yea, I can move this up to the top.